bin/dump-postgres.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DATE=$(date +"%Y-%m-%d-%H-%M")
+
+DUMPDIR=/home/noiapp/dump
+DESTFILE=$DUMPDIR/dump-$DATE.tar
+LATEST=$DUMPDIR/latest
+ls -t | tail -n +5 | xargs rm --
+docker exec noiapp_noiapp-database_1 bash -c 'pg_dump --create --format t -U noiapp'>$DESTFILE
```

Comment:
if the command execution fails for any reason, `DESTFILE` is created anyway.
You should check return code and, if the command is failed, drop `DESTFILE` and report failure w/o changing `LATEST`.

Reply:
I don't know how to do this, please help me

Reply:
something like:
```bash
docker exec noiapp_noiapp-database_1 bash -c 'pg_dump --create --format t -U noiapp' >$DESTFILE
RET=$?
if [ $RET -ne 0 ] ; then
  rm $DESTFILE
  echo "DB dump has failed, $DESTFILE not created." >&2 
  exit $RET
fi
```
not tested (I cannot do it by now) but you can give it a try by using a not-existing user as `pg_dump` argument

Reply:
thanks, it works! (tested with no container and with wrong user)